cmd/internal/planetscale_edge_database_test.go

```diff
@@ -38,7 +39,7 @@ func TestRead_CanPeekBeforeRead(t *testing.T) {
 	}
 
 	cc := clientConnectionMock{
-		syncFn: func(ctx context.Context, in *psdbconnect.SyncRequest, opts ...grpc.CallOption) (psdbconnect.Connect_SyncClient, error) {
+		syncFn: func(ctx context.Context, in *connect.Request[psdbconnect.SyncRequest]) (*connect.ServerStreamForClient[psdbconnect.SyncResponse], error) {
```

Comment:
I got real lost in the sauce here and gave up with the mocks. With grpc-go, the `Connect_SyncClient` was an interface, so it was simpler to mock, with `connect-go` these are a concrete type `*connect.ServerStreamForClient[T any]`, so I'm not sure how to mock this.